dotcom-rendering/src/web/components/Carousel.importable.tsx

```diff
@@ -505,9 +505,7 @@ export const Carousel = ({ heading, trails, onwardsSource, format }: Props) => {
 
 		const scrolled = current.scrollLeft + offset;
 
-		const nextOffset = offsets
-			.reverse()
-			.find((offset) => offset < scrolled);
+		const nextOffset = offsets.reverse().find((o) => o < scrolled);
```

Comment:
Fix a 'no shadow' error

Reply:
Could also drop the `reverse` with [`Array.prototype.findLast` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findLast)